core/include/drivers/zynqmp_csu.h

```diff
@@ -0,0 +1,33 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) Foundries Ltd. 2021
+ * Author: Jorge Ramirez <jorge@foundries.io>
+ */
+
+#ifndef ZYNQMP_CSU_H_
```

Comment:
We normally prefix with the path relative to the include directory, in this case I'd make that `__DRIVERS_ZYNQMP_CSU_H`

Reply:
ok